css/wee.module.less

```diff
@@ -16,9 +16,12 @@
 .wee-{{moduleName}}-responsive () when (@responsiveEnabled = true) {
 	.check () when not (@mobileLandscapeWidth = false) {
 		@media (min-width: (@mobileLandscapeWidth)) {
-			html {
-				font-family: '"2"';
+			.check () when ({{standalone}} = true) {
```

Comment:
So standalone means that the module is all by itself with no other modules or core CSS, therefore the breakpoint sizes are necessary to add?